app/helpers/application_helper.rb

```diff
@@ -80,4 +83,15 @@ def get_body_css_class(controller)
 
     "#{classes[controller]}"
   end
+
+  def alert_class(alert)
+    case alert
+    when 'error', 'alert'
+      'alert-danger'
+    when 'notice'
+      'alert-success'
+    else
+      'alert-info'
+    end
+  end
```

Comment:
@mllocs what do you think? specially this commit: https://github.com/coopdevs/timeoverflow/pull/361/commits/761c7ff9d38c74fae3a31a285387b36d3892e6c1 😄